src/validator.directive.js

```diff
@@ -133,7 +145,9 @@
         return $q.all([$validationProvider.getExpression(validator)(value, scope, element, attrs, validatorParam)])
           .then(function(data) {
             if (data && data.length > 0 && data[0]) return valid.success();
-            else return valid.error();
+            else if (attrs.validationGroup) {
+              if (!checkValidationGroup(scope, element, attrs, ctrl)) valid.error();
```

Comment:
I think that we don't have to rewrite the valid message if the group is already valid.